Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pythonPipeline {
   extras_require = 'dev'
   upload_dev_wheels = false
   pep440 = true
-  venv_rpm_deps = ['gcc-c++', 'kstart']
+  venv_rpm_deps = ['gcc-c++', 'kstart', 'krb5-devel']
```

Comment:
FYI:
I added the krb5-devel package, which provides `krb5-config`, because requests_gssapi needs it during its pip installation. I noticed that when the Jenkins build failed.
(requests_gssapi is used for Kerberos authentication)